tests/run-make/print-target-list/rmake.rs

```diff
@@ -0,0 +1,15 @@
+// Checks that all the targets returned by `rustc --print target-list` are valid
+// target specifications
+
+//@ needs-llvm-components: aarch64 arm avr bpf csky hexagon i386 i586 i686 loongarch64
+//@ needs-llvm-components: m68k mips msp430 nvptx64 powerpc riscv s390x sparc thumb wasm x86 xtensa
```

Comment:
Suggestion: not needed since I found out why Makefile wasn't failing

Reply:
Suggestion: we should revert the tidy changes since they're no longer necessary.